lms/djangoapps/teams/static/teams/js/spec/views/topics_spec.js

```diff
@@ -64,8 +47,25 @@ define([
             var requests = AjaxHelpers.requests(this),
                 topicsView = createTopicsView();
 
-            topicsView.collection.teamEvents.trigger('teams:update', { action: 'create' });
-            topicsView.render();
+            triggerUpdateEvent(topicsView);
+            AjaxHelpers.expectJsonRequestURL(
+                requests,
+                'api/teams/topics',
+                {
+                    course_id : 'my/course/id',
+                    page : '1',
+                    page_size : '5',  // currently the page size is determined by the size of the collection
+                    order_by : 'name'
+                }
+            );
+        });
+
+        it('refreshes the topics staff creates a team and then joins it', function() {
```

Comment:
Nit: I should change this to "refreshes the topics WHEN staff creates...".

This is the test case that covers the bug.